hevc_codec_registration.src.html

```diff
@@ -0,0 +1,212 @@
+    It describes, for HEVC (H.265), the (1) fully qualified codec strings,
+    (2) the codec-specific {{EncodedVideoChunk}}
+    [=EncodedVideoChunk/[[internal data]]=] bytes, (3) the
+    {{VideoDecoderConfig.description}} bytes, (4) the values of
+    {{EncodedVideoChunk}} [=EncodedVideoChunk/[[type]]=], and (5) the
+    codec-specific extensions to {{VideoEncoderConfig}}
+
+    The registration is not intended to include any information on whether a
+    codec format is encumbered by intellectual property claims. Implementers and
+    authors are advised to seek appropriate legal counsel in this matter if they
+    intend to implement or use a specific codec format. Implementers of
+    WebCodecs are not required to support the HEVC / H.265 codec.
+
+    This registration is non-normative.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: attribute
+        text: EncodedVideoChunkMetadata.decoderConfig; url: dom-encodedvideochunkmetadata-decoderconfig
+        for: EncodedVideoChunkType; text: key; url: dom-encodedvideochunktype-key
+        text: VideoDecoderConfig.description; url: dom-videodecoderconfig-description
+    type: dfn
+        for: EncodedVideoChunk; text: [[internal data]]; url: dom-encodedvideochunk-internal-data-slot
+        for: EncodedVideoChunk; text: [[type]]; url: dom-encodedvideochunk-type-slot
+        for: VideoEncoder; text: [[output callback]]; url: dom-videoencoder-output-callback-slot
+    type: interface
+        text: EncodedVideoChunk; url: encodedvideochunk
+        text: VideoEncoder; url: videoencoder
+    type: dictionary
+        text: VideoEncoderConfig; url: dictdef-videoencoderconfig
+        text: VideoDecoderConfig; url: dictdef-videodecoderconfig
+</pre>
+
+<pre class='biblio'>
+{
+  "ITU-T-REC-H.265": {
+    "href": "https://www.itu.int/rec/T-REC-H.265",
+    "title": "H.264 : Advanced video coding for generic audiovisual services",
+    "publisher": "ITU",
+    "date": "August 2021"
+  },
+  "iso14496-15": {
+    "href": "https://www.iso.org/standard/74429.html",
+    "title": "ISO/IEC 14496-15:2019 Information technology — Coding of audio-visual objects — Part 15: Carriage of network abstraction layer (NAL) unit structured video in the ISO base media file format",
+    "publisher": "ISO",
+    "date": "September 2019"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string begins with the prefix "hev1." or "hvc1.", with a suffix of
+four dot-separated fields as described in Section E.3 of [[iso14496-15]].
```

Comment:
I did not include hvc2, hev2, lhv1, lhe1, which are specific to L-HEVC and perhaps will be added at a later date.

Reply:
Title: line should use H.265, not H.264.